modules/persistent-storage-csi-migration-enable.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+[id="persistent-storage-csi-migration-enable_{context}"]
+= Enabling CSI automatic migration
+
+[NOTE]
+====
+Enabling CSI automatic migration drains, and then restarts, all nodes in the cluster in sequence. This might take some time.
+====
+
+.Procedure
+
+* Enable feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling features using feature gates]).
++
+[IMPORTANT]
+====
+After turning Technology Preview features on using feature gates, they cannot be turned off and cluster upgrades are prevented.
+====
++
+The following configuration example enables CSI automatic migration to both CSI drivers (AWS EBS and Cinder):
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: TechPreviewNoUpgrade <1>
+...
+----
+<1> Enables automatic migration for AWS EBS and Cinder.
```

Comment:
According to https://deploy-preview-31774--osdocs.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-enabling-features.html#nodes-cluster-enabling-features-about_nodes-cluster-enabling, the `TechPreviewNoUpgrade` feature set also includes other features.

1. Should we mention that here, adding something like ", among other features." or something?
2. Should that table in the link above be updated to include CSI migration as a feature that's enabled using it?

Reply:
1. I'm not sure that users really care about other features being enabled, and they will see that that is the case if they read the feature gates content.
2. Yes. This table is due to be updated: https://github.com/openshift/openshift-docs/pull/24132